environment_cpu.yml

```diff
@@ -7,6 +7,7 @@ channels:
   - conda-forge # pytest-mock
   - pytorch
 dependencies:
+  - python=3.10.10
```

Comment:
1. (Needed) Instead, can I get `python<3.11`? This is too restrictive.
2. (Nit) Can I get a comment that `onnxruntime` is the reason for the constraint? This will help me remember to check to get rid of it when possible.

Reply:
I changed it to python<3.11
this resulted in python 3.10.11 on my mac. No issues here.
Need to test with Ubuntu 23.04 and 22.04, so please wait with the merge.

Reply:
Have a look at the automatic tests that run, and consider contributing improvements?

Manual testing is a pain 😉

Reply:
Automated tests (which use Python 3.8, 3.9, 3.10 on Ubuntu) pass 👍🏻

Reply:
you are right, the automated tests pass and I will consider contributing improvements. 

For the moment, everything looks good. 
My manual tests with Ubuntu 23.04 and 22.04 passed.